includes/RequestWiki/SpecialRequestWiki.php

```diff
@@ -34,6 +34,7 @@ public function execute( $par ) {
 		$this->setParameter( $par );
 		$this->setHeaders();
 
+		$this->checkReadOnly();
```

Comment:
Wait, this isn't needed, checkExecutePermissions has checkReadOnly in it?

Reply:
Just define requiresWrite as true instead.

Reply:
Which is default so nevermind

Reply:
So is this redundant then?

Reply:
Yeah.

Reply:
That's funny. It didn't initially error me out when I set my own local test instance to read-only but did after I set this and even if I unset it. I'm guessing it was a momentary delay in something. Anyhow, something which does 100% work for a fact even in read-only is RequestWikiQueue so I'll focus on that instead